src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/Source/StaticLocals.vb

```diff
@@ -922,5 +922,172 @@ End Module
             CompileAndVerify(compilation)
         End Sub
 
+        <Fact>
+        <WorkItem(264417, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=264417")>
+        Public Sub InitializeWithAsNew_01()
```

Comment:
Consider naming the tests `InitializeStructureWithAsNew_` and `InitializeClassWithAsNew_` so it's clear what the difference is between the pairs of tests.